src/abstractions/platformUtils.service.ts

```diff
@@ -14,6 +14,9 @@ export abstract class PlatformUtilsService {
     isMacAppStore: () => boolean;
     analyticsId: () => string;
     isViewOpen: () => Promise<boolean>;
+    /**
+     * @deprecated This only ever returns null. Pull from your platform's storage using ConstantsService.vaultTimeoutKey
+     */
```

Comment:
is this true now, or will be based on your changes?

Reply:
This seems to be true today. Investigated all implementations of the `PlatformUtils` interface and no usage returned any value besides `null`. 

The second-half of that comment to use the new `ConstantsService` key will be from here moving forward.

Will keep an eye on this as I dive into each platform to make sure that statement remains true.